lichen_cli/src/main.rs

```diff
@@ -122,11 +122,11 @@ fn ask_rootfs(parts: &[SystemPartition]) -> color_eyre::Result<&SystemPartition>
 
 fn ask_filesystem() -> color_eyre::Result<String> {
     let variants = [
-        ("xfs", "XFS", "For reliability (journaling)"),
-        ("f2fs", "F2FS", "For flash storage (speed)"),
-        ("ext4", "Ext4", "Not recommended due to small hardlink limit"),
+        ("xfs", "XFS", "Recommended (fast with moss hardlink rollbacks)"),
+        ("f2fs", "F2FS", "Not Recommended (slower with moss hardlink rollbacks)"),
+        ("ext4", "EXT4", "Not Recommended (slower with moss hardlink rollbacks, limited rollback capacity)"),
```

Comment:
I've never seen ext4 capitalized, where does that come from?

Reply:
Fair point. In reality, they should probably all just have their first letter capitalised.

Reply:
Somehow XFS and F2FS look less weird. Definitely more weird with the first letter capitalized IMHO.